### ISSUE TITLE
Global Undo/Redo

### ISSUE THREAD

**Issue by Esshahn** [nurpax/petmate#132]
After working with Petmate for quite some time, I've made up my mind about the undo/redo implementation and I suggest to switch to a global (all screens) undo, not on a per screen basis.

When working with multiple screens and undoing a lot of steps, it always comes to a halt on the current active screen as the undo only works for the active screen.

I think it would make more sense to have the undo work for alle screens and removing a newly created screen should be part of the undo stack.

**Comment by nurpax** [comment]
Agreed. It’s a deep change but having screen ops undoable is already a great benefit

**Comment by nurpax** [comment]
FWIW, this is one of the things that should be considered should I ever add font editing into Petmate.  Right now, implementing undo for editing custom fonts would get really confusing.  If I put everything under the same undo stack, it's kinda obvious what undo/redo means for font bitmap edits.
